config/config.go

```diff
@@ -16,6 +15,7 @@ type config struct {
 	AppId         int    `json:"app_id"`
 	ApiHash       string `json:"api_hash"`
 	RedisUri      string `json:"redis_uri"`
+	RedisCloudUrl string `json:"rediscloud_url"`
```

Comment:
Adding a separate var for verified users than the common ones make the config env quite confusing.
How about removing `RedisUri` and `RedisPass` and just keeping the `RedisCloudUrl` for everyone?

Reply:
No ? It gives users more choices :/